redaction/redactor.go

```diff
@@ -321,7 +321,7 @@ func GetKeyValuesToRedact(logger shell.Logger, patterns []string, environment ma
 
 			if matched {
 				if len(varValue) < RedactLengthMin {
-					logger.Warningf("Value of %s below minimum length and will not be redacted", varName)
+					logger.Warningf("Value of %s below minimum length (%d bytes) and will not be redacted", varName, RedactLengthMin)
```

Comment:
Technically I _think_ the length limit is in Unicode [runes](https://go.dev/blog/strings), not bytes.
I think the 4-rune string “🔑🔑🔑🔑” would fall short of a `RedactLengthMin = 8`, even though it's 16 bytes.

Reply:
(but with “rune” being a Go-specific term, I guess we'd call them “characters”)

Reply:
`len("some string")` returns the number of bytes in a string, rather than the number of runes. if we wanted to count runes here (which wouldn't be unreasonable i think, it's unlikely but not out of the question that someone uses é in their password), we'd have use `utf8.RuneCountInString()` - see [this go playground](https://go.dev/play/p/dzWSRVQ4FPI)

Reply:
Ah — thanks for fact checking my fake news :)